### PR TITLE
[MIG][15.0] hr: Migration script

### PR DESCRIPTION
**Pull request by sang250399**
This PR
======
Migration done

### CODE REVIEW

**Review by gdgellatly — COMMENTED** [review]
Static review, not tested. lgtm

File: openupgrade_scripts/scripts/hr/15.0.1.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE hr_employee
+            SET departure_reason_id=CASE departure_reason
+                    WHEN 'fired' THEN % s
+                    WHEN 'resigned' THEN % s
+                    WHEN 'retired' THEN % s
+                END
+            WHERE departure_reason IN('fired', 'resigned', 'retired')
```
Comment (phamgiang2510) [inline openupgrade_scripts/scripts/hr/15.0.1.1/post-migration.py:35]: Reformat the code
```suggestion
            UPDATE hr_employee
            SET departure_reason_id = CASE departure_reason
                    WHEN 'fired' THEN %s
                    WHEN 'resigned' THEN %s
                    WHEN 'retired' THEN %s
                END
            WHERE departure_reason IN ('fired', 'resigned', 'retired')
```

**Comment by MiquelRForgeFlow** [comment]
Merged in https://github.com/OCA/OpenUpgrade/pull/3481.

**Pull request closed without merging**